components/counter/src/index.js

```diff
@@ -0,0 +1,8 @@
+import { AuroCounter } from './auro-counter.js';
+import { AuroCounterGroup } from './auro-counter-group.js';
+
+AuroCounter.register();
+AuroCounterGroup.register();
+
+// Enable TypeScript support
+export { AuroCounter, AuroCounterGroup };
```

Comment:
I like this exporting idea. @jason-capsule42 should we introduce this to other comps?

Reply:
I'd like to see auro-counter completed and look more at this before we start changing other components. 

I suggest you create an investigation ticket to weight this an potential improvement to other components after we finish the work already on our plate.

Reply:
This must have been a change added by @chrisfalaska since it's already on all the components. Sorry didn't realize sooner.